lib/pharos/phases/configure_master.rb

```diff
@@ -310,6 +326,18 @@ def upload_authentication_token_webhook_certs(webhook_config)
         @ssh.file(PHAROS_DIR + '/token_webhook/key.pem').write(File.open(File.expand_path(webhook_config[:user][:client_key]))) if webhook_config[:user][:client_key]
       end
 
+      def configure_secrets_encryption
+        # Generate keys
+        key1 = Base64.strict_encode64(SecureRandom.random_bytes(32))
+        key2 = Base64.strict_encode64(SecureRandom.random_bytes(32))
+        cluster_context['secrets_encryption'] = {
+          'key1' => key1,
+          'key2' => key2
+        }
+
+        @ssh.file(SECRETS_CFG_FILE).write(parse_resource_file('secrets/encryption-config.yml.erb', key1: key1, key2: key2))
```

Comment:
For something like plaintext secrets encryption keys in particular, it would be a good idea to keep this file non-world-readable on the master host.

The `/var/lib/etcd/member/` directory seems to be 0700 already.